src/cli/common.rs

```diff
@@ -56,13 +56,11 @@ pub(crate) enum Confirm {
     Advanced,
 }
 
-pub(crate) fn confirm_advanced(customized_install: &CustomizedInstall) -> Result<Confirm> {
+pub(crate) fn confirm_advanced(customized_install: &bool) -> Result<Confirm> {
```

Comment:
There's no point in passing `bool` by reference, just pass it directly here.

Reply:
Is that best practice in rust in general? pass reference only when necessary I mean

also done :)

Reply:
Yes -- in particular, `Copy` types (including `bool`, integer and floating point types) are typically passed by-value.

Reply:
Sweet as, going in the anki